src/modules/login/index.tsx

```diff
@@ -25,27 +69,142 @@ export default function LoginScreen() {
 
   return (
     <View style={styles.container}>
-      <Text style={styles.title}>LOGIN</Text>
-      <Button
-        onPress={openTabs}
-        title="Go to application"
-        color="#841584"
-      />
+      <View style={styles.header}>
+        <Image
+            source={require('../../../assets/images/logo.png')}
+        />
+      </View>
+      <View style={styles.footer}>
+        <View>
+          <TextInput
```

Comment:
You probably will reuse this `Input`. It's better to move the component out so you can import it in other screens.